_ml-commons-plugin/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title: About ML Commons 
+nav_order: 1
+has_children: false
+has_toc: false
+---
+
+# ML Commons plugin
+
+ML Commons for OpenSearch eases the development of machine learning features by providing a set of common machine learning (ML) algorithms through transport and REST API calls. Those calls choose the right nodes and resources for each ML request and monitors ML tasks to ensure uptime. This allows you to leverage existing open-source ML algorithms and reduce the effort required to develop new ML features. 
+
+Models trained through the ML Commons plugin support two types of algorithms: 
+
+- Model-based algorithms such kmeans or Linear Regression. To get the best results, make sure you train your model first, then use the model to apply predictions. Linear Regression is only supported 
```

Comment:
"Model-based algorithms such **as** kmeans and Linear Regression"

Missing some text after "Linear Regression is only supported"?